packages/plugins/src/plugins/eval/index.ts

```diff
@@ -76,13 +76,15 @@ export default function (
                 .compileScriptSync(code_to_run)
                 .runSync(context, { timeout });
         } catch (e) {
-            sendData(
-                false,
-                JSON.stringify(e.toString()),
-                JSON.stringify([]),
-                start,
-                Date.now()
-            );
+            if (typeof e === "object" && e) {
```

Comment:
What if it's not an object?

Reply:
As discussed, switching to `Object.prototype.toString.call(e)` in 4cdf7b731b8f364377d24c974fd755e9a627a1e8